docs/maps/temperaturemap-series.md

```diff
@@ -0,0 +1,50 @@
+Temperature map
```

Comment:
Should this file be deleted now that we moved the feature over to standard bubble?